srcpkgs/libnma/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'libnma'
+pkgname=libnma
+version=1.8.30
+revision=1
+build_style=meson
+build_helper="gir"
```

Comment:
you will need the "qemu" build helper too.

Reply:
I changed the `build_helper` value to `"gir qemu"`, but I'm still getting the same errors as before on the ARM architectures.